### PR TITLE
Clean up ALB routing capability information

### PR DESCRIPTION
**Pull request by ThanosBaskous**
See #468.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1047,14 +1047,13 @@ ALB
 ### ALB Tips
 
 -	Use ALBs to route to services that are hosted on shared clusters with dynamic port assignment (like ECS or Mesos).
--	ALBs support HTTP path-based routing (send HTTP requests for “/api/*” -> {target-group-1}, “/blog/*” -> {target group 2}).
+-	ALBs support [HTTP host-based routing](http://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-listeners.html#host-conditions) (send HTTP requests for “api.mydomain.com” -> {target-group-1}, “api.mydomain.com” -> {target group 2}) as well as [HTTP path-based routing](http://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-listeners.html#path-conditions) (send HTTP requests for “/api/&ast;” ->  {target-group-1}, “/blog/&ast;” -> {target group 2}).
```
Comment (jlevy) [inline README.md]: The first hosts should be e.g. api.mydomain.com and blog.mydomain.com right?

Comment (QuinnyPig) [inline README.md]: Yeah, I think the second occurance of 'api.mydomain.com' should instead be 'blog.mydomain.com' unless I misunderstand>

Comment (ThanosBaskous) [inline README.md]: Good catch, thanks - fixed.

**Comment by ThanosBaskous** [comment]
👍 issue with ALB routing option description fixed.

**Pull request merged**